README.md

```diff
@@ -40,13 +41,6 @@ If you find bugs or have problems with the CalDAV-Backend, you should ask the te
 * Internet Explorer 11
 * Safari 12.1+
 
-## Maintainers
-
-- [Christoph Wurst](https://github.com/ChristophWurst)
-- [Thomas Citharel](https://github.com/tcitworld)
-- [Richard Steinmetz](https://github.com/st3iny)
-- [and many more](https://github.com/nextcloud/calendar/graphs/contributors)
-
 If you’d like to join, just go through the [issue list](https://github.com/nextcloud/calendar/issues?utf8=✓&q=is%3Aopen+is%3Aissue+label%3A%22good+first+issue%22+) and fix some. :)   
```

Comment:
Maybe move this paragraph somewhere else now (below building ?)

Reply:
Let me actually drop this line until we have good first issues https://github.com/nextcloud/groupware/issues/16